### PR TITLE
Ddearaujo/de36130 close ssrf hole

### PR DESCRIPTION
**Pull request by Dan-DeAraujo**
Don't allow access to private or reserved IP ranges when making requests

### CODE REVIEW

File: doRequest.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Copyright (c) 2012 Desire2Learn Inc.
+ * Copyright (c) 2019 Desire2Learn Inc.
```
Comment (grant-cleary) [inline doRequest.php]: Does this actually need to change?

Comment (Dan-DeAraujo) [reply doRequest.php]: lol. dunno, i guess not... i just did because i happened to be there and notice it. I certainly don't care either way

Comment (grant-cleary) [reply doRequest.php]: What I mean is that I'm pretty sure the copyright term is supposed to start from the first year it's applicable, in which case this wouldn't change.

Comment (grant-cleary) [reply doRequest.php]: It's hard I guess when it comes to changing works like this, since "year of first publication" is a bit questionable.

**Comment by PeteLloyd** [comment]
LGTM

File: doRequest.php (hunk 1 above)
Comment (Dan-DeAraujo) [reply doRequest.php]: alrighty, i'll put it back then. 🙈

Comment (Dan-DeAraujo) [reply doRequest.php]: reverted in a62a9d6

**Pull request merged**